sharding-orchestration/sharding-orchestration-reg/sharding-orchestration-reg-nacos/src/main/java/org/apache/shardingsphere/orchestration/reg/nacos/NacosRegistryCenter.java

```diff
@@ -137,7 +138,7 @@ public void initLock(final String key) {
 
     @Override
     public boolean tryLock() {
-        return false;
+        return true;
     }

```

Comment:
the lock's methods should be used in ShardingKeyGenerator functions.
it used in `LeafSegmentKeyGenerator` and `LeafSnowflakeKeyGenerator`.

I think it don't return `true` value directly.

Reply:
yes. tryLock() should be implemented correctly for using Leaf×××××××Generator

Reply:
initLock() and tryRelease() also need to be implemented.